### ISSUE TITLE
Version of pytorch3d

### ISSUE THREAD

**Issue by weijielyu** [Jumpat/SegAnyGAussians#5]
Hi, thanks for this great work! I've been having problems installing the dependence. Could you please let me know which version of pytorch3d you used?

Thank you!

**Comment by Jumpat** [comment]
Hello, thanks for your interest! The pytorch3d we used is 0.7.1

The repo is still under construction. So there may be some unexpected problems. Thanks for your patience!

**Comment by norton-swz** [comment]
Hi, may I ask if you conducted the experiment in a Windows environment? I tried to install pytorch3D in Windows 11 but was unsuccessful

**Comment by Jumpat** [comment]
> Hi, may I ask if you conducted the experiment in a Windows environment? I tried to install pytorch3D in Windows 11 but was unsuccessful

Sorry but we haven't test the code on windows. Maybe you should check the repo of pytorch3D, where I think should has some guys with similar problems with you.

**Comment by vararth** [comment]
I am also facing the same issue in setting up the environment on Ubuntu 20.04 - 
```
(base) root@a100-instance-ajna:~/SegAnyGAussians# conda env create --file environment.yml 
Retrieving notices: ...working... done
Channels:
 - pytorch
 - conda-forge
 - defaults
Platform: linux-64
Collecting package metadata (repodata.json): done
Solving environment: failed

PackagesNotFoundError: The following packages are not available from current channels:

  - pytorch3d=0.7.1*

Current channels:

  - https://conda.anaconda.org/pytorch/linux-64
  - https://conda.anaconda.org/conda-forge/linux-64
  - https://repo.anaconda.com/pkgs/main/linux-64
  - https://repo.anaconda.com/pkgs/r/linux-64

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.
```

**Comment by dreamandgo-123** [comment]
> I am also facing the same issue in setting up the environment on Ubuntu 20.04 -
> 
> ```
> (base) root@a100-instance-ajna:~/SegAnyGAussians# conda env create --file environment.yml 
> Retrieving notices: ...working... done
> Channels:
>  - pytorch
>  - conda-forge
>  - defaults
> Platform: linux-64
> Collecting package metadata (repodata.json): done
> Solving environment: failed
> 
> PackagesNotFoundError: The following packages are not available from current channels:
> 
>   - pytorch3d=0.7.1*
> 
> Current channels:
> 
>   - https://conda.anaconda.org/pytorch/linux-64
>   - https://conda.anaconda.org/conda-forge/linux-64
>   - https://repo.anaconda.com/pkgs/main/linux-64
>   - https://repo.anaconda.com/pkgs/r/linux-64
> 
> To search for alternate channels that may provide the conda package you're
> looking for, navigate to
> 
>     https://anaconda.org
> 
> and use the search bar at the top of the page.
> ```

You can refer to this link: https://pypi.org/project/pytorch3d/0.7.1/#files;  for linux, the latest version of pytorch3d is 0.3.0
Maybe the server system of the authors is macosx.

**Comment by dreamandgo-123** [comment]
> So sorry for the late response @vararth ! Thanks @dreamandgo-123 for the help. Our system version is ubuntu 20.04.1 but the PyTorch3d version is exactly 0.7.1. Maybe you can check the official repo of pytorch3d for more information.

Yes, @Jumpat is right, thank you very much. I tried more times and found that the right repository of pytorch3d is [here](https://anaconda.org/pytorch3d/pytorch3d/files?page=6), and you @vararth  may choose the proper version according to your conda environment. The installation command is "conda install xxx.bz2"（Thanks to [here](https://zhuanlan.zhihu.com/p/627637604)）. Best wishes!

**Comment by vararth** [comment]
Thank you both for the help, I got the environment set up!
I just removed the Pytorch3d requirement from the environment.yml and instead got the correct version from github and compiled it manually :) - fortunately worked smoothly after that!

**Comment by zratclif** [comment]
Hello, I have also run into the problem where the channels I currently am using do not contain the version of pytorch3d required. The answers above seem helpful I just can't quite get them to work, does anyone have any suggestions?

**Comment by Jumpat** [comment]
Hi, what problems did you encounter? I think the issues of pytorch3d may be helpful.

> Hello, I have also run into the problem where the channels I currently am using do not contain the version of pytorch3d required. The answers above seem helpful I just can't quite get them to work, does anyone have any suggestions?

**Comment by zratclif** [comment]
I finally was able to figure it out, thank you.
